### PR TITLE
Sensors, SetProp, and SetRule

### PR DESCRIPTION
**Pull request by TripleToxic**
Contained features from previous pull request: ``currentAmmoType``, ``velocityX``, ``velocityY``, and new features:
- Setters for ``velocityX`` and ``velocityY`` in ``SetProp``
- A setter for ``dragMultiplier`` in ``SetRule``

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

edit: fixed description

### CODE REVIEW

**Comment by TripleToxic** [comment]
Oops, wrong description

File: core/src/mindustry/entities/comp/UnitComp.java [hunk 1]
```diff
@@ -214,6 +214,8 @@ public double sense(LAccess sensor){
             case ammoCapacity -> type.ammoCapacity;
             case x -> World.conv(x);
             case y -> World.conv(y);
+            case velocityX -> vel.x;
+            case velocityY -> vel.y;
```
Comment (Anuken) [inline core/src/mindustry/entities/comp/UnitComp.java:218]: This returns velocity in world units, but logic operates in tiles. It must be converted, like speed.

File: core/assets/bundles/bundle.properties [hunk 2]
```diff
@@ -2434,6 +2434,7 @@ lenum.shootp = Shoot at a unit/building with velocity prediction.
 lenum.config = Building configuration, e.g. sorter item.
 lenum.enabled = Whether the block is enabled.
 
+laccess.currentammotype = Item/liquid that is corresponded to the current ammo of a turret.
```
Comment (Anuken) [inline core/assets/bundles/bundle.properties:2437]:
```suggestion
laccess.currentammotype = Current ammo item/liquid of a turret.
```

File: core/src/mindustry/world/blocks/defense/turrets/Turret.java [hunk 3]
```diff
@@ -293,7 +293,7 @@ public void control(LAccess type, Object p1, double p2, double p3, double p4){
         }
 
         @Override
-        public double sense(LAccess sensor){
+        public double sense(LAccess sensor) {
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/defense/turrets/Turret.java:296]:
```suggestion
        public double sense(LAccess sensor){
```

**Pull request merged**